pages/join.vue

```diff
@@ -11,7 +11,7 @@
 
             <div class="mt-12">
               <p class="title">We can't do this work alone.</p>
-              <p>We've grown from a pair of researchers from Stanford University and the University of Waterloo into an interdisciplinary team of more than 400 researchers, software engineers, public health and security experts.</p>
+              <p>We've grown from a pair of researchers from Stanford University and the University of Waterloo into an interdisciplinary team of more than 400 researchers, software engineers, public health experts and security experts.</p>
```

Comment:
Not sure about the redundancy here, though I agree that the sentence as it stands isn't great. @jcolla-holla what do you think of the following:

```suggestion
              <p>We've grown from a pair of researchers from Stanford University and the University of Waterloo into an interdisciplinary team of more than 400 researchers, software engineers, public health professionals. and security experts.</p>
```

Reply:
+1 for "public health professionals, and security experts."